<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alarmMainWindow</class>
 <widget class="QMainWindow" name="alarmMainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTalarm</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/new/icons/Icons/Clock.png</normaloff>:/new/icons/Icons/Clock.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgba(31,53,114, 0.6);
color: #FFFFFF; /* 白色文字 */

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget { 
    /*background-image: url(:/centerWidget_bg2.jpg); */
	background-image:url(:/calendar_bg.jpg);

    background-position: center;
    background-size: cover;
}

QCheckBox:checked, QRadioButton:checked {
    background-color: #fedd2a; /* 选中时的背景颜色 */
	color: #1A2B49;
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="6">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QCalendarWidget" name="calendarWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QCalendarWidget {
    background-color: rgba(26, 43, 73, 180); /* 半透明深蓝色背景 */
    color: #FFFFFF; /* 白色文字 */
    border: 1px solid rgba(62, 80, 98, 150); /* 半透明边框 */
    border-radius: 10px; /* 圆角边框 */
    font-family: Arial, sans-serif; /* 设置字体为 Arial, sans-serif */
    font-size: 14px; /* 设置字体大小为14px */
}

QCalendarWidget QWidget#qt_calendar_navigationbar {
    background-color: rgba(45, 63, 90, 150); /* 导航栏背景颜色，半透明深蓝色 */
    color: #FFFFFF; /* 导航栏文字颜色为白色 */
    border: none; /* 无边框 */
}

QCalendarWidget QToolButton {
    height: 35px; /* 设置按钮高度为35px */
    width: 120px; /* 设置按钮宽度为120px */
    color: #FFFFFF; /* 按钮文字颜色为白色 */
    font-size: 16px; /* 设置按钮文字大小为16px */
    icon-size: 24px, 24px; /* 设置按钮图标尺寸为24x24px */
    background-color: rgba(62, 80, 98, 150); /* 按钮背景颜色，半透明深蓝色 */
    border: none; /* 无边框 */
    border-radius: 5px; /* 设置按钮圆角边框为5px */
}

QCalendarWidget QToolButton::hover {
    background-color: rgba(74, 96, 122, 150); /* 鼠标悬停时的按钮背景颜色，半透明蓝色 */
}

QCalendarWidget QToolButton::pressed {
    background-color: rgba(88, 106, 133, 150); /* 按钮被按下时的背景颜色，半透明蓝色 */
}

QCalendarWidget QAbstractItemView {
    background-color: rgba(255, 255, 255, 80); /* 半透明白色背景 */
    alternate-background-color: rgba(255, 255, 255, 60); /* 交替行的半透明白色背景 */
    color: #FFFFFF; /* 日历项文字颜色为白色 */
    border: none; /* 无边框 */
}

QCalendarWidget QAbstractItemView:enabled {
    font-size: 14px; /* 设置日历项文字大小为14px */
    background-color: rgba(255, 255, 255, 80); /* 半透明白色背景 */
    selection-background-color: #fedd2a; /* 选中日期的背景颜色为#fedd2a */
    selection-color: #1A2B49; /* 选中日期的文字颜色为深蓝色，确保在黄色背景上可读 */
    border: none; /* 无边框 */
}

QCalendarWidget QAbstractItemView:disabled {
    color: #777777; /* 禁用日期的文字颜色为灰色 */
}

QCalendarWidget QAbstractItemView:focus {
    border: 1px solid rgba(62, 80, 98, 150); /* 获得焦点时的边框颜色，半透明蓝色 */
}

QCalendarWidget QHeaderView::section {
    background-color: rgba(45, 63, 90, 150); /* 日期头部背景颜色，半透明深蓝色 */
    color: #F7D268; /* 日期头部文字颜色为金色 */
    border: 1px solid rgba(62, 80, 98, 150); /* 日期头部的边框颜色，半透明蓝色 */
}

QCalendarWidget QAbstractItemView:vertical-header {
    background-color: rgba(45, 63, 90, 150); /* 垂直日期头部背景颜色，半透明深蓝色 */
    color: #F7D268; /* 垂直日期头部文字颜色为金色 */
}
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="6" colspan="3">
     <widget class="QPushButton" name="TestBtn">
      <property name="text">
       <string>测试</string>
      </property>
     </widget>
    </item>
    <item row="9" column="8">
     <widget class="QDialogButtonBox" name="listAlmBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ignore|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QCheckBox" name="chkBastard">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string extracomment="Force math questions to disable alarm"/>
      </property>
      <property name="text">
       <string>解决问题来关闭闹钟</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QCheckBox" name="chkSounds">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>使用文件</string>
      </property>
     </widget>
    </item>
    <item row="6" column="8">
     <widget class="QLabel" name="txtSoundPath">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" rowspan="2" colspan="8">
     <widget class="QListWidget" name="listWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255,255,255,0.5)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>调整音量</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="6" colspan="3">
     <widget class="QSlider" name="VolumeSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="6" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>当前时间</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <item row="2" column="0" colspan="2">
        <widget class="myClockWidgetFinal" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(0,0,0,0)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="Clock">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF; /* 白色文字 */</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="dateLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QCheckBox" name="chkCustom">
      <property name="text">
       <string>设置日期:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QDateEdit" name="CustEdit"/>
    </item>
    <item row="5" column="3">
     <widget class="QCheckBox" name="chkOnshot">
      <property name="text">
       <string>仅本周</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QCheckBox" name="chkSun">
      <property name="text">
       <string>周日</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="chkSat">
      <property name="text">
       <string>周六</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="chkFri">
      <property name="text">
       <string>周五</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QTimeEdit" name="timeEdit">
      <property name="displayFormat">
       <string>hh:mm:ss:AP</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="lblTime">
      <property name="text">
       <string>     时间：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QCheckBox" name="chkThurs">
      <property name="text">
       <string>周四</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="chkWed">
      <property name="text">
       <string>周三</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="chkTues">
      <property name="styleSheet">
       <string notr="true">selection-background-color: #fedd2a;</string>
      </property>
      <property name="text">
       <string>周二</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="chkMon">
      <property name="text">
       <string>周一</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QLabel" name="label_2">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>设置闹钟</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="6">
     <widget class="QLabel" name="lblPoem">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件 (&amp;f)</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于 (&amp;u)</string>
    </property>
    <addaction name="actionAbout_QT"/>
    <addaction name="actionAbout_QTalam"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>退出 (&amp;q）</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>关于QT (&amp;a)</string>
   </property>
  </action>
  <action name="actionAbout_QTalam">
   <property name="text">
    <string>关于时钟模块 (&amp;Q)</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>设置 (&amp;s)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myClockWidgetFinal</class>
   <extends>QWidget</extends>
   <header>myclockwidgetfinal.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
